smalig/grammar.yaml

```diff
@@ -68,22 +68,22 @@
   format: "B|A|op"
   format_id: "12x"
   syntax: "move-wide vA, vB"
-  args_info: "A: destination register pair (4 bits), B: source register pair (4 bits)"
+  args_info: "A: destination register-pair (4 bits), B: source register-pair (4 bits)"
   short_desc: "Move the contents of one register-pair to another."
-  long_desc: "Move the value of the (vB, vB+1) register-pair to the (vA, vA+1) register-pair. Both register pairs must be in the first 16 register range (0-15)."
-  note: "It is legal to move from vN to either vN-1 or vN+1, so implementations must arrange for both halves of a register pair to be read before anything is written."
-  example: "" #TODO: Add example
-  example_desc: ""
+  long_desc: "Move the value of the (vB, vB+1) register-pair to the (vA, vA+1) register-pair. Both register-pairs must be in the first 16 register range (0-15)."
+  note: "It is legal to move from vN to either vN-1 or vN+1, so implementations must arrange for both halves of a register-pair to be read before anything is written."
+  example: "0411 - move-wide v1, v2"
+  example_desc: "Moves the value from register-pair (v2, v3) to register-pair (v1, v2)."
 
 - opcode: "05"
   name: "move-wide/from16"
   format: "AA|op BBBB"
   format_id: "22x"
   syntax: "move-wide/from16 vAA, vBBBB"
-  args_info: "A: destination register pair (8 bits), B: source register pair (16 bits)"
+  args_info: "A: destination register-pair (8 bits), B: source register-pair (16 bits)"
   short_desc: "Move the contents of one register-pair to another."
   long_desc: "Move the value (long/double) of the (vBBBB, vBBBB+1) register-pair to the (vAA, vAA+1) register-pair. vB must be in the 64k register range (0-65535) while vA is one of the first 256 registers (0-255)."
-  note: "It is legal to move from vN to either vN-1 or vN+1, so implementations must arrange for both halves of a register pair to be read before anything is written."
+  note: "It is legal to move from vN to either vN-1 or vN+1, so implementations must arrange for both halves of a register-pair to be read before anything is written."
   example: "0516 0000 - move-wide/from16 v22, v0"
   example_desc: "Moves v0 into v22."
```

Comment:
This is incorrect on my part. Could you please correct it if possible? Otherwise, conflicts will arise.